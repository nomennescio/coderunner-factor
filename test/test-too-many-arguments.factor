! Copyright 2019 nomennescio

USING: tools.testest ;
IN: tests

: run-tests ( -- )

<{ 0 0 -> 0 }>

;

MAIN: run-tests

