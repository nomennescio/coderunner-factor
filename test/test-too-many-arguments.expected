
<FAILED::>Test Failed
Expected :
0
, but got :
0
0
